src/main/java/crazypants/enderio/conduit/gui/GuiExternalConnectionSelector.java

```diff
@@ -80,11 +84,21 @@ private void go(EnumFacing dir) {
     player.openGui(EnderIO.instance, GuiHandler.GUI_ID_EXTERNAL_CONNECTION_BASE + dir.ordinal(), player.worldObj, loc.x, loc.y, loc.z);
   }
 
+  protected String getBlockNameForDirection(EnumFacing direction) {
+    Block b = cb.getBundleWorldObj().getBlockState(cb.getLocation().getLocation(direction).getBlockPos()).getBlock();
+    if (b != null && b != Blocks.air) {
```

Comment:
Please use world.isAirBlock(), otherwise things like the LightNode will show up here, too. I'd also like conduits filtered out, too.